### DISCUSSION TITLE
Update user card design

### DISCUSSION THREAD

**Discussion by stanriders** [ppy/osu#22118] [Ideas] [answered]
Current user card design is quite dated and uninformative, [the new one](https://www.figma.com/file/ocltATjJqWQZBravhPuqjB/UI%2FPlayer-Card?node-id=0%3A1) seems much better as well as ready for implementation.

- [ ] Implement user group badges (#22123)
- [ ] Update the layout
- [ ] Implement expanded version
- [ ] Put additional data in like ranks? #2510

I'd like to have a go at implementing the new design myself

**Comment by peppy** [comment, marked as answer]
Could you please post this as an issue rather than a discussion, then assign yourself?

**Reply by stanriders** [reply]
Sure, I will escalate it into an issue, just wasn't sure of the process
